pom.xml

```diff
@@ -150,7 +150,7 @@
                             <title>Kafka Connect Elasticsearch</title>
                             <documentationUrl>https://docs.confluent.io/${project.version}/connect/connect-elasticsearch/docs/index.html</documentationUrl>
                             <description>
-                                The Elasticsearch connector allows moving data from Kafka to Elasticsearch 2.x, 5.x, 6.x, and 7.x. It writes data from a topic in Kafka to an index in Elasticsearch and all data for a topic have the same type.
+                                The Elasticsearch connector allows moving data from Kafka to Elasticsearch 2.x, 5.x, 6.x, 7.x, and 8.x. The support for Elasticsearch depends on which version the connector is running. Newer versions of the connector may not support all previous versions of ES. ES 8.x was introduced from version 14.0.0. It writes data from a topic in Kafka to an index in Elasticsearch and all data for a topic have the same type.
```

Comment:
```suggestion
                                The Elasticsearch connector moves data from Kafka to Elasticsearch 2.x, 5.x, 6.x, 7.x, and 8.x. Elasticsearch version support depends the connector version. Newer connector versions may not support older versions of Elasticsearch. Elasticsearch 8.x support was introduced with connector version 14.0.0. This connector version writes data from a topic in Kafka to an index in Elasticsearch and all data for the topic have the same type.
```

Reply:
@sp-gupta some minor suggestions for rewording

Reply:
@tobydrake7 done

Reply:
@sp-gupta you may want to expand on this part `add data for a topic have the same type`  does this mean that the connector keeps all datatypes from the topic consistent when it moves it to Elasticsearch? might be good to add a little more to this or add an extra explanatory sentence about this 👍

Reply:
@tobydrake7 Thanks, I have updated it. Please review.